README.md

```diff
@@ -27,6 +27,30 @@ While running locally, you might want to emulate mining on the blockchain.
 Use the command 
 * `$ yarn dev:forge`
 
+## Technical Details
+
+### Create Secure Account Flow
+* User inputs Date of Birth and National Identity Document
+* Secure HashId is Generated with those info, using SHA-3 Algorithm
+* Eligibility Authority (EA) checks if HashID matches the Eligible HashID list they own and authorizes (or not) user to place its vote
+
+### Register PIN and Activate Blockchain Account
+* To the HashId is added a Random Value
+* With those values a Passphrase is generated (locally)
+* The generated passphrase is used to create the User Keys (1 private Keys and 1 public)
+* User then creates a 5 digit PIN, that is used to encript the passphrase and store it locally
+
+### Encripted Vote
+* A request to the Blockchain Node is performed, so user can Activate his Blockchain account
+* When a new block is forged (about 4mins in production environment), user is allowed to place its vote
+* The voting transaction is encripted, and a small fee is paid for it. The value to pay the fee is provided to the user by EA
+* 
+
+### Results Reveal
+* When ellection event is finished, the EA discloses it's privateKey
```

Comment:
When ellection event is finished, the EA discloses it's Voting campaign privateKey